src/VisualStudio/Core/Def/Preview/FileChange.cs

```diff
@@ -235,15 +236,22 @@ private static IHierarchicalDifferenceCollection ComputeDiffSpans(ITextDifferenc
         //       current way of just using text differ has its own issue, and using syntax differ in compiler that are for incremental parser
         //       has its own drawbacks.
 
+        var differenceOptions = new StringDifferenceOptions()
+        {
+            DifferenceType = StringDifferenceTypes.Line,
+        };
```

Comment:
nit.  make static readonly member of this type.

Reply:
Done!